common/source/docs/common-blheli32-passthru.rst

```diff
@@ -30,14 +30,19 @@ Depending on the ESC, BLHeli/BLHeli_S/BLHeli32/AM32 provides the following featu
 Pass-Through Support
 --------------------
 
-..note:: This feature is only available on NON IOMCU outputs. Autopilots which have an IOMCU co-processsor (usually marked as having "MAIN" outputs from the IOMCU and "AUX" outputs from the main cpu) will not pass-through on those outputs. Use this features on "AUX" or "FMU" outputs with DShot capability.
+.. note:: 
+   This feature is only available on NON IOMCU outputs. Autopilots which have an IOMCU co-processsor (usually marked as having "MAIN" outputs from the IOMCU and "AUX" outputs from the main cpu) will not pass-through on those outputs.
+   Use this feature on "AUX" or "FMU" outputs with DShot capability. Be mindful that using AUX outputs for flight-critical controls removes the benefits of IOMCU.
```

Comment:
```suggestion
   Use this feature on "AUX" or "FMU" outputs with DShot capability. For Plane, Rover and Sub, be mindful that using AUX outputs for vehicle-critical controls removes the benefits of and IOMCU. For Copter, IOMCU redundancy is not effective since it has no "MANUAL" control mode.
```

Reply:
LGTM, should say "an IOMCU"

Reply:
@Ryanf55 are you going to update? or do you want me to finish it off?

Reply:
ping @Ryanf55 sitll want to pursue this??